src/SearchableEntity.php

```diff
@@ -61,11 +61,9 @@ public function getIndexUid(): string
      */
     public function getSearchableArray(): array
     {
-        /** @var \Doctrine\ORM\Mapping\ClassMetadataInfo&\Doctrine\Persistence\Mapping\ClassMetadata $metadata */
-        $metadata = $this->entityMetadata;
-
         $context = [
-            'fieldsMapping' => $metadata->fieldMappings,
+            /* @phpstan-ignore-next-line */
+            'fieldsMapping' => $this->entityMetadata->fieldMappings,
```

Comment:
Did not succeed to make PHPStan work with this despite the code is working.

Reply:
I opened this in the meantime to avoid losing time fixing this.
https://github.com/meilisearch/meilisearch-symfony/issues/158